x/minfee/module.go

```diff
@@ -52,66 +53,63 @@ func (AppModule) IsOnePerModuleType() {}
 
 // Name returns the minfee module's name.
 func (AppModule) Name() string {
-	return ModuleName
+	return types.ModuleName
 }
 
 // RegisterLegacyAminoCodec registers the blob module's types on the LegacyAmino codec.
 func (AppModule) RegisterLegacyAminoCodec(_ *codec.LegacyAmino) {}
 
-// RegisterInterfaces registers interfaces and implementations of the blob module.
-func (AppModule) RegisterInterfaces(_ cdctypes.InterfaceRegistry) {}
+// RegisterInterfaces registers interfaces and implementations of the minfee module.
+func (AppModule) RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
+	types.RegisterInterfaces(registry)
+}
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the module.
 func (am AppModule) RegisterGRPCGatewayRoutes(_ client.Context, _ *runtime.ServeMux) {}
 
 // RegisterServices registers module services.
-func (am AppModule) RegisterServices(registrar grpc.ServiceRegistrar) error {
-	RegisterQueryServer(registrar, NewQueryServerImpl(am.paramsKeeper))
-	return nil
+func (am AppModule) RegisterServices(cfg module.Configurator) {
+	types.RegisterMsgServer(cfg.MsgServer(), am.minfeeKeeper)
+	types.RegisterQueryServer(cfg.QueryServer(), am.minfeeKeeper)
+
+	m := keeper.NewMigrator(am.minfeeKeeper)
+	if err := cfg.RegisterMigration(types.ModuleName, 1, m.MigrateParams); err != nil {
```

Comment:
Should be 2 here right? I'm on mobile and forgot if it is from version or to version 😅

Reply:
consensus version was `1` before, this PR is changing it to `2`